ingest-app/src/cmr/ingest/api/ingest/collections.clj

```diff
@@ -0,0 +1,69 @@
+(ns cmr.ingest.api.ingest.collections
```

Comment:
Moved this code, but not yet pushed -- it's a fairly big shift, so I'm re-running tests before I push it up.